examples/Alert.types.tsx

```diff
@@ -0,0 +1,29 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { Alert } from '@itwin/itwinui-react';
+
+export default () => {
+  return (
+    <>
+      {(['positive', 'warning', 'negative'] as const).map((type) => (
+        <Alert.Wrapper
+          key={type}
+          type={type}
+          style={{ minWidth: 'min(100%, 350px)' }}
+        >
+          <Alert.Icon />
+          <Alert.Message>
+            This is a {type} alert
+            <Alert.Action onClick={() => console.log('Clicked more info!')}>
+              Learn more
+            </Alert.Action>
+          </Alert.Message>
+          <Alert.CloseButton onClick={() => console.log('CLOSED')} />
+        </Alert.Wrapper>
+      ))}
+    </>
+  );
```

Comment:
it would be nice to have some gap between these
![](https://github.com/iTwin/iTwinUI/assets/9084735/bac543a9-f65d-4253-9503-6c219331ef92)

for now we can add a flex wrapper:
```suggestion
import { Alert, Flex } from '@itwin/itwinui-react';

export default () => {
  return (
    <Flex flexDirection='column'>
      {(['positive', 'warning', 'negative'] as const).map((type) => (
        <Alert.Wrapper key={type} type={type}>
          <Alert.Icon />
          <Alert.Message>
            This is a {type} alert
            <Alert.Action onClick={() => console.log('Clicked more info!')}>
              Learn more
            </Alert.Action>
          </Alert.Message>
          <Alert.CloseButton onClick={() => console.log('CLOSED')} />
        </Alert.Wrapper>
      ))}
    </Flex>
  );
```